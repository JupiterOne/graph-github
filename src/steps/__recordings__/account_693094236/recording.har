{
  "log": {
    "_recordingName": "account",
    "creator": {
      "comment": "persister:JupiterOneIntegationFSPersister",
      "name": "Polly.JS",
      "version": "5.1.1"
    },
    "entries": [
      {
        "_id": "7f2926ac1ad814d2dfcaf9525a63bf36",
        "_order": 0,
        "cache": {},
        "request": {
          "bodySize": 0,
          "cookies": [],
          "headers": [
            {
              "_fromType": "array",
              "name": "accept",
              "value": "application/vnd.github.machine-man-preview+json"
            },
            {
              "_fromType": "array",
              "name": "user-agent",
              "value": "jupiter-integration-github octokit-rest.js/18.11.4 octokit-core.js/3.5.1 Node.js/14.17.3 (linux; x64)"
            },
            {
              "_fromType": "array",
              "name": "authorization",
              "value": "Bearer [REDACTED]"
            },
            {
              "_fromType": "array",
              "name": "content-length",
              "value": "0"
            },
            {
              "_fromType": "array",
              "name": "accept-encoding",
              "value": "gzip,deflate"
            },
            {
              "_fromType": "array",
              "name": "connection",
              "value": "close"
            },
            {
              "name": "host",
              "value": "api.github.com"
            }
          ],
          "headersSize": 812,
          "httpVersion": "HTTP/1.1",
          "method": "POST",
          "queryString": [],
          "url": "https://api.github.com/app/installations/17214088/access_tokens"
        },
        "response": {
          "bodySize": 348,
          "content": {
            "mimeType": "application/json; charset=utf-8",
            "size": 348,
            "text": "{\"token\":\"[REDACTED]\",\"expires_at\":\"2050-12-31T18:09:20Z\",\"permissions\":{\"members\":\"read\",\"organization_administration\":\"read\",\"organization_secrets\":\"read\",\"actions\":\"read\",\"contents\":\"read\",\"environments\":\"read\",\"issues\":\"read\",\"metadata\":\"read\",\"pull_requests\":\"read\",\"secrets\":\"read\"},\"repository_selection\":\"all\"}"
          },
          "cookies": [],
          "headers": [
            {
              "name": "server",
              "value": "GitHub.com"
            },
            {
              "name": "date",
              "value": "Fri, 28 Jan 2022 18:46:09 GMT"
            },
            {
              "name": "content-type",
              "value": "application/json; charset=utf-8"
            },
            {
              "name": "content-length",
              "value": "348"
            },
            {
              "name": "cache-control",
              "value": "public, max-age=60, s-maxage=60"
            },
            {
              "name": "vary",
              "value": "Accept, Accept-Encoding, Accept, X-Requested-With"
            },
            {
              "name": "etag",
              "value": "\"044f36ac305be9f2d993e67c4b74e822ba08c8cb3b8526f27eb7c9d8dd42a559\""
            },
            {
              "name": "x-github-media-type",
              "value": "github.v3; param=machine-man-preview; format=json"
            },
            {
              "name": "access-control-expose-headers",
              "value": "ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset"
            },
            {
              "name": "access-control-allow-origin",
              "value": "*"
            },
            {
              "name": "strict-transport-security",
              "value": "max-age=31536000; includeSubdomains; preload"
            },
            {
              "name": "x-frame-options",
              "value": "deny"
            },
            {
              "name": "x-content-type-options",
              "value": "nosniff"
            },
            {
              "name": "x-xss-protection",
              "value": "0"
            },
            {
              "name": "referrer-policy",
              "value": "origin-when-cross-origin, strict-origin-when-cross-origin"
            },
            {
              "name": "content-security-policy",
              "value": "default-src 'none'"
            },
            {
              "name": "x-github-request-id",
              "value": "F948:4187:997B6D:13EAC7E:61F439F0"
            },
            {
              "name": "connection",
              "value": "close"
            }
          ],
          "headersSize": 1072,
          "httpVersion": "HTTP/1.1",
          "redirectURL": "",
          "status": 201,
          "statusText": "Created"
        },
        "startedDateTime": "2022-01-28T18:46:08.677Z",
        "time": 347,
        "timings": {
          "blocked": -1,
          "connect": -1,
          "dns": -1,
          "receive": 0,
          "send": 0,
          "ssl": -1,
          "wait": 347
        }
      },
      {
        "_id": "90e92246ca868687b35a21f1736a0e7d",
        "_order": 0,
        "cache": {},
        "request": {
          "bodySize": 0,
          "cookies": [],
          "headers": [
            {
              "_fromType": "array",
              "name": "accept",
              "value": "application/vnd.github.v3+json"
            },
            {
              "_fromType": "array",
              "name": "user-agent",
              "value": "jupiter-integration-github octokit-rest.js/18.11.4 octokit-core.js/3.5.1 Node.js/14.17.3 (linux; x64)"
            },
            {
              "_fromType": "array",
              "name": "authorization",
              "value": "Bearer [REDACTED]"
            },
            {
              "_fromType": "array",
              "name": "accept-encoding",
              "value": "gzip,deflate"
            },
            {
              "_fromType": "array",
              "name": "connection",
              "value": "close"
            },
            {
              "name": "host",
              "value": "api.github.com"
            }
          ],
          "headersSize": 761,
          "httpVersion": "HTTP/1.1",
          "method": "GET",
          "queryString": [],
          "url": "https://api.github.com/app/installations/17214088"
        },
        "response": {
          "bodySize": 1363,
          "content": {
            "mimeType": "application/json; charset=utf-8",
            "size": 1363,
            "text": "{\"id\":17214088,\"account\":{\"login\":\"Kei-Institute\",\"id\":84923503,\"node_id\":\"MDEyOk9yZ2FuaXphdGlvbjg0OTIzNTAz\",\"avatar_url\":\"https://avatars.githubusercontent.com/u/84923503?v=4\",\"gravatar_id\":\"\",\"url\":\"https://api.github.com/users/Kei-Institute\",\"html_url\":\"https://github.com/Kei-Institute\",\"followers_url\":\"https://api.github.com/users/Kei-Institute/followers\",\"following_url\":\"https://api.github.com/users/Kei-Institute/following{/other_user}\",\"gists_url\":\"https://api.github.com/users/Kei-Institute/gists{/gist_id}\",\"starred_url\":\"https://api.github.com/users/Kei-Institute/starred{/owner}{/repo}\",\"subscriptions_url\":\"https://api.github.com/users/Kei-Institute/subscriptions\",\"organizations_url\":\"https://api.github.com/users/Kei-Institute/orgs\",\"repos_url\":\"https://api.github.com/users/Kei-Institute/repos\",\"events_url\":\"https://api.github.com/users/Kei-Institute/events{/privacy}\",\"received_events_url\":\"https://api.github.com/users/Kei-Institute/received_events\",\"type\":\"Organization\",\"site_admin\":false},\"repository_selection\":\"all\",\"access_tokens_url\":\"https://api.github.com/app/installations/17214088/access_tokens\",\"repositories_url\":\"https://api.github.com/installation/repositories\",\"html_url\":\"https://github.com/organizations/Kei-Institute/settings/installations/17214088\",\"app_id\":117431,\"app_slug\":\"jupiterone-kei-institute\",\"target_id\":84923503,\"target_type\":\"Organization\",\"permissions\":{\"issues\":\"read\",\"actions\":\"read\",\"members\":\"read\",\"secrets\":\"read\",\"contents\":\"read\",\"metadata\":\"read\",\"environments\":\"read\",\"pull_requests\":\"read\",\"organization_secrets\":\"read\",\"organization_administration\":\"read\"},\"events\":[],\"created_at\":\"2021-05-27T15:42:53.000Z\",\"updated_at\":\"2022-01-11T17:09:24.000Z\",\"single_file_name\":null,\"has_multiple_single_files\":false,\"single_file_paths\":[],\"suspended_by\":null,\"suspended_at\":null}"
          },
          "cookies": [],
          "headers": [
            {
              "name": "server",
              "value": "GitHub.com"
            },
            {
              "name": "date",
              "value": "Fri, 28 Jan 2022 18:46:09 GMT"
            },
            {
              "name": "content-type",
              "value": "application/json; charset=utf-8"
            },
            {
              "name": "cache-control",
              "value": "public, max-age=60, s-maxage=60"
            },
            {
              "name": "vary",
              "value": "Accept, Accept-Encoding, Accept, X-Requested-With"
            },
            {
              "name": "etag",
              "value": "W/\"4662b5a4aa4ae852addefe7cfe4b37830f617deae0434f60ddc8e5bc9a704433\""
            },
            {
              "name": "x-github-media-type",
              "value": "github.v3; format=json"
            },
            {
              "name": "access-control-expose-headers",
              "value": "ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset"
            },
            {
              "name": "access-control-allow-origin",
              "value": "*"
            },
            {
              "name": "strict-transport-security",
              "value": "max-age=31536000; includeSubdomains; preload"
            },
            {
              "name": "x-frame-options",
              "value": "deny"
            },
            {
              "name": "x-content-type-options",
              "value": "nosniff"
            },
            {
              "name": "x-xss-protection",
              "value": "0"
            },
            {
              "name": "referrer-policy",
              "value": "origin-when-cross-origin, strict-origin-when-cross-origin"
            },
            {
              "name": "content-security-policy",
              "value": "default-src 'none'"
            },
            {
              "name": "x-github-request-id",
              "value": "F949:1619:CBD0E1:197A78A:61F439F1"
            },
            {
              "name": "connection",
              "value": "close"
            }
          ],
          "headersSize": 1078,
          "httpVersion": "HTTP/1.1",
          "redirectURL": "",
          "status": 200,
          "statusText": "OK"
        },
        "startedDateTime": "2022-01-28T18:46:09.056Z",
        "time": 316,
        "timings": {
          "blocked": -1,
          "connect": -1,
          "dns": -1,
          "receive": 0,
          "send": 0,
          "ssl": -1,
          "wait": 316
        }
      },
      {
        "_id": "22e4c882794d943e8a6f0961702d5000",
        "_order": 0,
        "cache": {},
        "request": {
          "bodySize": 506,
          "cookies": [],
          "headers": [
            {
              "name": "content-type",
              "value": "application/json"
            },
            {
              "name": "accept",
              "value": "application/json"
            },
            {
              "name": "user-agent",
              "value": "jupiterone-graph-github"
            },
            {
              "name": "authorization",
              "value": "Bearer [REDACTED]"
            },
            {
              "name": "host",
              "value": "api.github.com"
            }
          ],
          "headersSize": 228,
          "httpVersion": "HTTP/1.1",
          "method": "POST",
          "postData": {
            "mimeType": "application/json",
            "params": [],
            "text": "{\"query\":\"query ($login: String!) {\\n    organization(login: $login) {\\n        id\\n        ... organizationFields\\n      }\\n... rateLimit\\n  }\\n\\nfragment organizationFields on Organization {\\n    login\\n    name\\n    createdAt\\n    updatedAt\\n    description\\n    email\\n    databaseId\\n    isVerified\\n    location\\n    websiteUrl\\n    url\\n  }\\n\\nfragment rateLimit on Query {\\n    rateLimit {\\n      limit\\n      cost\\n      remaining\\n      resetAt\\n    }\\n  }\",\"variables\":{\"login\":\"Kei-Institute\"}}"
          },
          "queryString": [],
          "url": "https://api.github.com/graphql"
        },
        "response": {
          "bodySize": 481,
          "content": {
            "mimeType": "application/json; charset=utf-8",
            "size": 481,
            "text": "{\"data\":{\"organization\":{\"id\":\"MDEyOk9yZ2FuaXphdGlvbjg0OTIzNTAz\",\"login\":\"Kei-Institute\",\"name\":\"Kei-Institute\",\"createdAt\":\"2021-05-27T15:21:12Z\",\"updatedAt\":\"2022-01-06T17:48:33Z\",\"description\":\"Here's my test description\",\"email\":\"email@email.com\",\"databaseId\":84923503,\"isVerified\":false,\"location\":\"Albania\",\"websiteUrl\":\"www.goclickatesturl.com\",\"url\":\"https://github.com/Kei-Institute\"},\"rateLimit\":{\"limit\":5000,\"cost\":1,\"remaining\":4961,\"resetAt\":\"2022-01-28T19:44:46Z\"}}}"
          },
          "cookies": [],
          "headers": [
            {
              "name": "server",
              "value": "GitHub.com"
            },
            {
              "name": "date",
              "value": "Fri, 28 Jan 2022 18:46:09 GMT"
            },
            {
              "name": "content-type",
              "value": "application/json; charset=utf-8"
            },
            {
              "name": "content-length",
              "value": "481"
            },
            {
              "name": "x-github-media-type",
              "value": "github.v4"
            },
            {
              "name": "x-ratelimit-limit",
              "value": "5000"
            },
            {
              "name": "x-ratelimit-remaining",
              "value": "4961"
            },
            {
              "name": "x-ratelimit-reset",
              "value": "1643399086"
            },
            {
              "name": "x-ratelimit-used",
              "value": "39"
            },
            {
              "name": "x-ratelimit-resource",
              "value": "graphql"
            },
            {
              "name": "access-control-expose-headers",
              "value": "ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset"
            },
            {
              "name": "access-control-allow-origin",
              "value": "*"
            },
            {
              "name": "strict-transport-security",
              "value": "max-age=31536000; includeSubdomains; preload"
            },
            {
              "name": "x-frame-options",
              "value": "deny"
            },
            {
              "name": "x-content-type-options",
              "value": "nosniff"
            },
            {
              "name": "x-xss-protection",
              "value": "0"
            },
            {
              "name": "referrer-policy",
              "value": "origin-when-cross-origin, strict-origin-when-cross-origin"
            },
            {
              "name": "content-security-policy",
              "value": "default-src 'none'"
            },
            {
              "name": "vary",
              "value": "Accept-Encoding, Accept, X-Requested-With"
            },
            {
              "name": "x-github-request-id",
              "value": "F94A:4B95:8FC1D2:12E9E06:61F439F1"
            },
            {
              "name": "connection",
              "value": "close"
            }
          ],
          "headersSize": 1040,
          "httpVersion": "HTTP/1.1",
          "redirectURL": "",
          "status": 200,
          "statusText": "OK"
        },
        "startedDateTime": "2022-01-28T18:46:09.396Z",
        "time": 332,
        "timings": {
          "blocked": -1,
          "connect": -1,
          "dns": -1,
          "receive": 0,
          "send": 0,
          "ssl": -1,
          "wait": 332
        }
      }
    ],
    "pages": [],
    "version": "1.2"
  }
}
