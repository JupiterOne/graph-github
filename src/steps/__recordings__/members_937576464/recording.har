{
  "log": {
    "_recordingName": "members",
    "creator": {
      "comment": "persister:JupiterOneIntegationFSPersister",
      "name": "Polly.JS",
      "version": "5.1.1"
    },
    "entries": [
      {
        "_id": "06d6593fc4a07c98683e9f0972b1c5e0",
        "_order": 0,
        "cache": {},
        "request": {
          "bodySize": 0,
          "cookies": [],
          "headers": [
            {
              "_fromType": "array",
              "name": "accept",
              "value": "application/vnd.github.machine-man-preview+json"
            },
            {
              "_fromType": "array",
              "name": "user-agent",
              "value": "jupiter-integration-github octokit-rest.js/18.12.0 octokit-core.js/3.5.1 Node.js/16.14.2 (darwin; x64)"
            },
            {
              "_fromType": "array",
              "name": "authorization",
              "value": "[REDACTED]"
            },
            {
              "_fromType": "array",
              "name": "content-length",
              "value": "0"
            },
            {
              "_fromType": "array",
              "name": "accept-encoding",
              "value": "gzip,deflate"
            },
            {
              "_fromType": "array",
              "name": "connection",
              "value": "close"
            },
            {
              "name": "host",
              "value": "api.github.com"
            }
          ],
          "headersSize": 813,
          "httpVersion": "HTTP/1.1",
          "method": "POST",
          "queryString": [],
          "url": "https://api.github.com/app/installations/23522163/access_tokens"
        },
        "response": {
          "bodySize": 102,
          "content": {
            "mimeType": "application/json; charset=utf-8",
            "size": 102,
            "text": "{\"message\":\"A JSON web token could not be decoded\",\"documentation_url\":\"https://docs.github.com/rest\",\"token\":\"[REDACTED]\",\"expires_at\":\"2050-12-31T18:09:20Z\"}"
          },
          "cookies": [],
          "headers": [
            {
              "name": "server",
              "value": "GitHub.com"
            },
            {
              "name": "date",
              "value": "Fri, 02 Sep 2022 15:59:41 GMT"
            },
            {
              "name": "content-type",
              "value": "application/json; charset=utf-8"
            },
            {
              "name": "content-length",
              "value": "102"
            },
            {
              "name": "x-github-media-type",
              "value": "github.v3; param=machine-man-preview; format=json"
            },
            {
              "name": "access-control-expose-headers",
              "value": "ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset"
            },
            {
              "name": "access-control-allow-origin",
              "value": "*"
            },
            {
              "name": "strict-transport-security",
              "value": "max-age=31536000; includeSubdomains; preload"
            },
            {
              "name": "x-frame-options",
              "value": "deny"
            },
            {
              "name": "x-content-type-options",
              "value": "nosniff"
            },
            {
              "name": "x-xss-protection",
              "value": "0"
            },
            {
              "name": "referrer-policy",
              "value": "origin-when-cross-origin, strict-origin-when-cross-origin"
            },
            {
              "name": "content-security-policy",
              "value": "default-src 'none'"
            },
            {
              "name": "vary",
              "value": "Accept-Encoding, Accept, X-Requested-With"
            },
            {
              "name": "x-github-request-id",
              "value": "F4B9:689B:57D06:D196E:6312286D"
            },
            {
              "name": "connection",
              "value": "close"
            }
          ],
          "headersSize": 939,
          "httpVersion": "HTTP/1.1",
          "redirectURL": "",
          "status": 401,
          "statusText": "Unauthorized"
        },
        "startedDateTime": "2022-09-02T15:59:40.909Z",
        "time": 160,
        "timings": {
          "blocked": -1,
          "connect": -1,
          "dns": -1,
          "receive": 0,
          "send": 0,
          "ssl": -1,
          "wait": 160
        }
      }
    ],
    "pages": [],
    "version": "1.2"
  }
}
