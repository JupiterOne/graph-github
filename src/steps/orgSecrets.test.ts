jest.setTimeout(30000);

import {
  Recording,
  executeStepWithDependencies,
} from '@jupiterone/integration-sdk-testing';
import { buildStepTestConfig } from '../../test/config';
import { setupGithubRecording } from '../../test/recording';
import { Steps } from '../constants';

let recording: Recording;
afterEach(async () => {
  await recording.stop();
});

test('fetchOrgSecrets exec handler', async () => {
  recording = setupGithubRecording({
    directory: __dirname,
    name: 'orgSecrets',
  });

  const stepConfig = buildStepTestConfig(Steps.FETCH_ORG_SECRETS);
  const stepResults = await executeStepWithDependencies(stepConfig);

  expect(stepResults).toMatchStepMetadata(stepConfig);
});
