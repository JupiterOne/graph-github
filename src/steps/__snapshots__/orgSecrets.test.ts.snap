// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fetchOrgSecrets exec handler 1`] = `
Object {
  "collectedEntities": Array [
    Object {
      "_class": Array [
        "Secret",
      ],
      "_key": "GitHub_Org_JupiterOne-Sandbox_Secret_ORG_SECRET_11",
      "_rawData": Array [
        Object {
          "name": "default",
          "rawData": Object {
            "created_at": "2023-02-09T23:10:00Z",
            "name": "ORG_SECRET_11",
            "repos": Array [
              Object {
                "_key": "R_kgDOI8BHIg",
                "databaseId": 599803682,
                "name": "literate-waffle",
              },
              Object {
                "_key": "R_kgDOI2pL-A",
                "databaseId": 594168824,
                "name": "reimagined-barnacle",
              },
              Object {
                "_key": "R_kgDOI2pNdw",
                "databaseId": 594169207,
                "name": "ubiquitous-umbrella",
              },
            ],
            "selected_repositories_url": "https://api.github.com/orgs/JupiterOne-Sandbox/actions/secrets/ORG_SECRET_11/repositories",
            "updated_at": "2023-02-09T23:10:00Z",
            "visibility": "selected",
          },
        },
      ],
      "_type": "github_org_secret",
      "createdOn": 1675984200000,
      "displayName": "ORG_SECRET_11",
      "name": "ORG_SECRET_11",
      "selectedRepositoriesLink": "https://api.github.com/orgs/JupiterOne-Sandbox/actions/secrets/ORG_SECRET_11/repositories",
      "updatedOn": 1675984200000,
      "visibility": "selected",
      "webLink": "https://github.com/organizations/JupiterOne-Sandbox/settings/secrets/actions/ORG_SECRET_11",
    },
    Object {
      "_class": Array [
        "Secret",
      ],
      "_key": "GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_rawData": Array [
        Object {
          "name": "default",
          "rawData": Object {
            "created_at": "2023-01-27T19:07:35Z",
            "name": "SECRET_ONE",
            "repos": Array [
              Object {
                "_key": "R_kgDOHNYdzQ",
                "databaseId": 483794381,
                "name": "archived-web-testing-playground",
              },
              Object {
                "_key": "R_kgDOIudGfA",
                "databaseId": 585582204,
                "name": "test-repo1",
              },
              Object {
                "_key": "R_kgDOIudewQ",
                "databaseId": 585588417,
                "name": ".github",
              },
              Object {
                "_key": "R_kgDOIwifvQ",
                "databaseId": 587767741,
                "name": "test-repo2",
              },
              Object {
                "_key": "R_kgDOIx9bTw",
                "databaseId": 589257551,
                "name": "austin-test",
              },
              Object {
                "_key": "R_kgDOI2pIIA",
                "databaseId": 594167840,
                "name": "intentionally-vulnerable-golang-project",
              },
              Object {
                "_key": "R_kgDOI2pLFA",
                "databaseId": 594168596,
                "name": "RepoWithPages",
              },
              Object {
                "_key": "R_kgDOI2pL-A",
                "databaseId": 594168824,
                "name": "reimagined-barnacle",
              },
              Object {
                "_key": "R_kgDOI2pNdw",
                "databaseId": 594169207,
                "name": "ubiquitous-umbrella",
              },
              Object {
                "_key": "R_kgDOI8BHIg",
                "databaseId": 599803682,
                "name": "literate-waffle",
              },
              Object {
                "_key": "R_kgDOJjTBjA",
                "databaseId": 640991628,
                "name": "ops-platform-jobs",
              },
              Object {
                "_key": "R_kgDOJyTpow",
                "databaseId": 656730531,
                "name": "integration-deployments-test",
              },
              Object {
                "_key": "R_kgDOJ54ckg",
                "databaseId": 664673426,
                "name": "oauth-apps-service-test",
              },
            ],
            "updated_at": "2023-02-09T23:11:31Z",
            "visibility": "all",
          },
        },
      ],
      "_type": "github_org_secret",
      "createdOn": 1674846455000,
      "displayName": "SECRET_ONE",
      "name": "SECRET_ONE",
      "selectedRepositoriesLink": undefined,
      "updatedOn": 1675984291000,
      "visibility": "all",
      "webLink": "https://github.com/organizations/JupiterOne-Sandbox/settings/secrets/actions/SECRET_ONE",
    },
  ],
  "collectedRelationships": Array [
    Object {
      "_class": "HAS",
      "_fromEntityKey": "O_kgDOB0VI3Q",
      "_key": "O_kgDOB0VI3Q|has|GitHub_Org_JupiterOne-Sandbox_Secret_ORG_SECRET_11",
      "_toEntityKey": "GitHub_Org_JupiterOne-Sandbox_Secret_ORG_SECRET_11",
      "_type": "github_account_has_org_secret",
      "displayName": "HAS",
    },
    Object {
      "_class": "USES",
      "_fromEntityKey": "R_kgDOI8BHIg",
      "_key": "R_kgDOI8BHIg|uses|GitHub_Org_JupiterOne-Sandbox_Secret_ORG_SECRET_11",
      "_toEntityKey": "GitHub_Org_JupiterOne-Sandbox_Secret_ORG_SECRET_11",
      "_type": "github_repo_uses_org_secret",
      "displayName": "USES",
    },
    Object {
      "_class": "USES",
      "_fromEntityKey": "R_kgDOI2pL-A",
      "_key": "R_kgDOI2pL-A|uses|GitHub_Org_JupiterOne-Sandbox_Secret_ORG_SECRET_11",
      "_toEntityKey": "GitHub_Org_JupiterOne-Sandbox_Secret_ORG_SECRET_11",
      "_type": "github_repo_uses_org_secret",
      "displayName": "USES",
    },
    Object {
      "_class": "USES",
      "_fromEntityKey": "R_kgDOI2pNdw",
      "_key": "R_kgDOI2pNdw|uses|GitHub_Org_JupiterOne-Sandbox_Secret_ORG_SECRET_11",
      "_toEntityKey": "GitHub_Org_JupiterOne-Sandbox_Secret_ORG_SECRET_11",
      "_type": "github_repo_uses_org_secret",
      "displayName": "USES",
    },
    Object {
      "_class": "HAS",
      "_fromEntityKey": "O_kgDOB0VI3Q",
      "_key": "O_kgDOB0VI3Q|has|GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_toEntityKey": "GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_type": "github_account_has_org_secret",
      "displayName": "HAS",
    },
    Object {
      "_class": "USES",
      "_fromEntityKey": "R_kgDOHNYdzQ",
      "_key": "R_kgDOHNYdzQ|uses|GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_toEntityKey": "GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_type": "github_repo_uses_org_secret",
      "displayName": "USES",
    },
    Object {
      "_class": "USES",
      "_fromEntityKey": "R_kgDOIudGfA",
      "_key": "R_kgDOIudGfA|uses|GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_toEntityKey": "GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_type": "github_repo_uses_org_secret",
      "displayName": "USES",
    },
    Object {
      "_class": "USES",
      "_fromEntityKey": "R_kgDOIudewQ",
      "_key": "R_kgDOIudewQ|uses|GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_toEntityKey": "GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_type": "github_repo_uses_org_secret",
      "displayName": "USES",
    },
    Object {
      "_class": "USES",
      "_fromEntityKey": "R_kgDOIwifvQ",
      "_key": "R_kgDOIwifvQ|uses|GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_toEntityKey": "GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_type": "github_repo_uses_org_secret",
      "displayName": "USES",
    },
    Object {
      "_class": "USES",
      "_fromEntityKey": "R_kgDOIx9bTw",
      "_key": "R_kgDOIx9bTw|uses|GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_toEntityKey": "GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_type": "github_repo_uses_org_secret",
      "displayName": "USES",
    },
    Object {
      "_class": "USES",
      "_fromEntityKey": "R_kgDOI2pIIA",
      "_key": "R_kgDOI2pIIA|uses|GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_toEntityKey": "GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_type": "github_repo_uses_org_secret",
      "displayName": "USES",
    },
    Object {
      "_class": "USES",
      "_fromEntityKey": "R_kgDOI2pLFA",
      "_key": "R_kgDOI2pLFA|uses|GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_toEntityKey": "GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_type": "github_repo_uses_org_secret",
      "displayName": "USES",
    },
    Object {
      "_class": "USES",
      "_fromEntityKey": "R_kgDOI2pL-A",
      "_key": "R_kgDOI2pL-A|uses|GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_toEntityKey": "GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_type": "github_repo_uses_org_secret",
      "displayName": "USES",
    },
    Object {
      "_class": "USES",
      "_fromEntityKey": "R_kgDOI2pNdw",
      "_key": "R_kgDOI2pNdw|uses|GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_toEntityKey": "GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_type": "github_repo_uses_org_secret",
      "displayName": "USES",
    },
    Object {
      "_class": "USES",
      "_fromEntityKey": "R_kgDOI8BHIg",
      "_key": "R_kgDOI8BHIg|uses|GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_toEntityKey": "GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_type": "github_repo_uses_org_secret",
      "displayName": "USES",
    },
    Object {
      "_class": "USES",
      "_fromEntityKey": "R_kgDOJjTBjA",
      "_key": "R_kgDOJjTBjA|uses|GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_toEntityKey": "GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_type": "github_repo_uses_org_secret",
      "displayName": "USES",
    },
    Object {
      "_class": "USES",
      "_fromEntityKey": "R_kgDOJyTpow",
      "_key": "R_kgDOJyTpow|uses|GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_toEntityKey": "GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_type": "github_repo_uses_org_secret",
      "displayName": "USES",
    },
    Object {
      "_class": "USES",
      "_fromEntityKey": "R_kgDOJ54ckg",
      "_key": "R_kgDOJ54ckg|uses|GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_toEntityKey": "GitHub_Org_JupiterOne-Sandbox_Secret_SECRET_ONE",
      "_type": "github_repo_uses_org_secret",
      "displayName": "USES",
    },
  ],
  "encounteredTypes": Array [
    "github_account",
    "github_repo",
    "github_account_owns_repo",
    "github_org_secret",
    "github_account_has_org_secret",
    "github_repo_uses_org_secret",
  ],
  "numCollectedEntities": 2,
  "numCollectedRelationships": 18,
}
`;
